src/ImageSharp/Formats/Jpeg/GolangPort/Components/Decoder/Bytes.cs

```diff
@@ -58,10 +58,12 @@ internal struct Bytes : IDisposable
         /// <returns>The bytes created</returns>
         public static Bytes Create(MemoryManager memoryManager)
```

Comment:
No need to pass the parameter here now or to the InputProcessor.

Reply:
Thanks for fixing it!